index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-setup-and-mobile-version-skeleton</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="name">Carlos Herver</div>
+    <div class="button"><img src="assets/Union.png" alt="Hamburger button"> </div>
+  </header>
+  <main>
+    <div class="intro-text">
+      <div class="primary-text">Hey There. I'm Carlos</div>
+      <div class="supporting-text">I'm a Software Developer</div>
+      <div class="Description-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</div>
+    </div>
+    <div class="social-media">
+      <div class="icon github"><img src="assets/github.svg" alt="github.com icon"></div>
+      <div class="icon linkedin"><img src="assets/Linkedin-icon.svg" alt="linkedin.com icon"></div>
+      <div class="icon angel-list"><img src="assets/angel-list.svg" alt="angel icon"></div>
+      <div class="icon twitter"><img src="assets/twitter.svg" alt="twitter.com icon"></div>
+      <div class="icon medium"><img src="assets/medium.svg" alt="medium.com icon"></div>
```

Comment:
It would be nice to put the social media icons in a `ul` as the parent container and use `li` to list each icon. This makes the HTML code more semantic. 🙏